docs/data/material/getting-started/templates/landing-page/LandingPage.js

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+
+import CssBaseline from '@mui/material/CssBaseline';
+import Box from '@mui/material/Box';
+import Divider from '@mui/material/Divider';
+import { ThemeProvider, createTheme } from '@mui/material/styles';
+import ToggleButton from '@mui/material/ToggleButton';
+import ToggleButtonGroup from '@mui/material/ToggleButtonGroup';
+
+import AutoAwesomeRoundedIcon from '@mui/icons-material/AutoAwesomeRounded';
+import AndroidRoundedIcon from '@mui/icons-material/AndroidRounded';
+
+import AppAppBar from './components/AppAppBar';
+import Hero from './components/Hero';
+import LogoCollection from './components/LogoCollection';
+import Highlights from './components/Highlights';
+import Pricing from './components/Pricing';
+import Features from './components/Features';
+import Testimonials from './components/Testimonials';
+import FAQ from './components/FAQ';
+import Footer from './components/Footer';
+
+import getLPTheme from './getLPTheme';
+
+const defaultTheme = createTheme({});
+
+function ToggleCustomTheme({ showCustomTheme, toggleCustomTheme }) {
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        width: '100dvw',
+        position: 'fixed',
+        bottom: 24,
+      }}
+    >
+      <ToggleButtonGroup
+        color="primary"
+        value={showCustomTheme}
+        exclusive
+        onChange={toggleCustomTheme}
+        aria-label="Platform"
+        sx={{
+          backgroundColor: 'background.default',
+          '& .Mui-selected': {
+            pointerEvents: 'none',
+          },
+        }}
+      >
+        <ToggleButton value>
+          <AutoAwesomeRoundedIcon sx={{ fontSize: '20px', mr: 1 }} />
+          Custom theme
+        </ToggleButton>
+        <ToggleButton value={false}>
+          <AndroidRoundedIcon sx={{ fontSize: '20px', mr: 1 }} />
+          Default theme
+        </ToggleButton>
+      </ToggleButtonGroup>
+    </Box>
+  );
+}
+
+ToggleCustomTheme.propTypes = {
+  showCustomTheme: PropTypes.shape({
+    valueOf: PropTypes.func.isRequired,
+  }).isRequired,
+  toggleCustomTheme: PropTypes.func.isRequired,
+};
```

Comment:
Should we remove the proptypes? It's not for everyone especially people who use TypeScript.

Reply:
It's from `docs:typescript:formatted`, isn't it? Also, this is in the `js.` file, users who use Typescript will use the `.tsx` file, won't they? 

Anyway, I leave the decision to you @siriwatknp :)